Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -872,7 +872,8 @@ public void ResetActionState(int actionIndex, InputActionPhase toPhase = InputAc
             // Wipe state.
             actionState->phase = toPhase;
             actionState->controlIndex = kInvalidIndex;
-            actionState->bindingIndex = memory.actionBindingIndices[memory.actionBindingIndicesAndCounts[actionIndex]];
+            var idx = memory.actionBindingIndicesAndCounts[actionIndex];
+            actionState->bindingIndex = memory.actionBindingIndices != null ? memory.actionBindingIndices[idx] : 0;
```

Comment:
Would it even be smart to set bindingIndex to -1 to indicate it's not a valid index? 
It would allow doing (actionState->bindingIndex >= 0) as an indirect null-check, but if other indices are zero it makes sense to keep it at zero for consistency

Reply:
This is a very good question to be honest. I'm surprised it's not been set to -1 before. I should if that's safe to change in order not to break some stuff somewhere, this code is kinda interconnected quite a bit.

Reply:
Awesome, it was just a thought and not something I consider a must-have if it adds complexity